codemods/1.x-2.x/__testfixtures__/operators-flow.output.js

```diff
@@ -14,7 +14,5 @@ export default [
   },
 
   // Wait 200ms, then continue chain
-  wait(200, [
-    doSomething
-  ])
+  wait(200)
```

Comment:
But you need the `doSomething` here, after wait from how I understand the output...